### DISCUSSION TITLE
How to celebrate the EddieHub Anniversary?

### DISCUSSION THREAD

**Discussion by ashwinexe** [EddieHubCommunity/support#1758] [Ideas] [answered]
EddieHub GitHub community was born **June 3rd 2020** and we need you help as to how are we going to celebrate its First Anniversary!

Drop your ideas below 👇

**Comment by eddiejaoude** [comment]
Oh interesting 🎉

**Reply by ghost** [reply]
EddieHub Quiz? Or a voice session? Or maybe something online. Or even a new video about EddieHub?

**Comment by kaiwalyakoparkar** [comment]
We can do the following things:
- Gaming session
- Video chat with hubbers
- Open source sessions for beginners

**Reply by ghost** [reply]
> Video chat 

😭

**Reply by ghost** [reply]
> Gaming session

🥳

**Reply by ashwinexe** [reply]
Video chat while gaming 😆

**Reply by ghost** [reply]
> Open source sessions for beginners

💪

**Reply by ghost** [reply]
> Video chat while gaming

Why is everyone up for video 😭

**Reply by kaiwalyakoparkar** [reply]
@krishdevdb Why? You don't like video calls?

**Reply by ghost** [reply]
> @krishdevdb Why? You don't like video calls?

Ya must know the public speaking fear, but mine ois video speaking feat 😓

**Comment by eddiejaoude** [comment, marked as answer]
Great suggestions everyone 👍 

I am keen to do something different, information but still fun.

I have an idea I am working on:
- 1 year EddieHub
- 20k celebration
- Live stream workshop on building "something" 🚀 
- Guest speakers
- Special give aways 🎉
